### PR TITLE
View headers owa

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5352dac659/how-to/)

**Comment by cyrichardson** [comment]
@DonNateR Suggested edits here: 
[view-and-read-email-headers-in-owa_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2210640/view-and-read-email-headers-in-owa_cyrEdits.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-14a385e242/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cf6ce74ef8/how-to/).

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+This article describes how to view message headers in the Outlook Web App.
+
+### Prerequisites
+
+- **Applies to:** Administrator and User
+- **Difficulty:** Moderate
+- **Time needed:** Approximately 5 minutes
+- **Tools required:**  Outlook Web App access
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest adding a slash at the end of this link.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### View headers in the Outlook Web App
+
+1. Log in to your mailbox at [apps.rackspace.com](apps.rackspace.com).
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest changing link to https://apps.rackspace.com/.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    <img src="{% asset_path exchange/view-and-read-email-headers-in-owa/header_window.png %}" />
+
+    **Note:** If the **Message details** box is blank, click **Close** and repeat steps 2-4. The message details should populate.
+
+You have successfully viewed the message headers in the Outlook Web App.
+
+### Understanding email headers
+
+The following header is an example of a *spoofed* message. If you suspect that you have received a spoofing email, see [Email spoofing explained](/how-to/email-spoofing-best-practices) for more information.
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest adding a closing slash at the end of this link.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- **Return-Path:** Like the **Reply-To:** address, this is where return mail is sent. Spammers can easily fake a return path. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
+- **Message-ID** A unique identifier assigned to a message. The **Message-ID** is useful for diagnosing a duplicate email issue. If you compare the **Message-ID** for multiple emails, and the IDs match, you know those messages are duplicates.
+- **X-Originating-Ip:** The IP address of the computer that sent the message. While this is slightly more difficult to fake, it is still possible. The originating IP address is typically the most reliable information about where the message actually came from. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
+
+### References
+
+- [Email spoofing explained](/how-to/email-spoofing-best-practices)
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest adding a closing slash to the end of each of these links.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    Received:	from yourdomainexample.com (localhost.localdomain [000.00.00.0]) by apps.net (Postfix) with for <boss@yourdomainexample.com>; Tue, 26 Sep 2017 14:52:40 -0400 (EDT)
+    Received:	by apps.rackspace.com (Authenticated sender: spoofer@yourdomainexample.com, from: assistant@yourdomainexample.com) with HTTP; Tue, 26 Sep 2017 14:52:40 -0400 (EDT)
+    Date:	Tue, 26 Sep 2017 14:52:40 -0400 (EDT)
+    Subject:	Send $$$
+    From:	"Assistant" <assistant@yourdomainexample.com>
+    To:	boss@yourdomainexample.com
+    Reply-To:	spoofer@scam.com
+    Message-ID:	<12345867.91012345@apps.rackspace.com>
+
+- **From:** Displays the message sender. Spammers can easily fake sender addresses. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest adding a closing slash at the end of this link.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+- **Subject:** The topic of the message as indicated by the sender.
+- **Date:** The date and time when the email message was composed.
+- **To:** Displays the addresses listed in the **To:** and **CC:** fields. Headers do not show any addresses that were included in the **BCC:** field, as these addresses were intended to remain private.
+- **Received:** Displays a sequential list of computer and servers that received this message, the time they received this message, and the final destination of the message. **Received** appears many times in a message header and should be read from bottom to top, as the first recipient is at the bottom of the header.
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest making "computer" plural.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+- **Reply-To:** Determines which email address is auto-populated when you click the reply button to reply to an email in your email client. Spammers can easily fake **Reply-To** addresses. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
+- **Return-Path:** Like the **Reply-To:** address, this is where return mail is sent. Spammers can easily fake a return path. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
+- **Message-ID** A unique identifier assigned to a message. The **Message-ID** is useful for diagnosing a duplicate email issue. If you compare the **Message-ID** for multiple emails, and the IDs match, you know those messages are duplicates.
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: 1. Suggest adding a colon (:) after the name of the field for consistency. 
2. It looks like some in-text references to fields (in the middle of sentences) have the colon and some don't. Suggest making this consistent.

3. Suggest removing comma after "emails".

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+- **From:** Displays the message sender. Spammers can easily fake sender addresses. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
+- **Subject:** The topic of the message as indicated by the sender.
+- **Date:** The date and time when the email message was composed.
+- **To:** Displays the addresses listed in the **To:** and **CC:** fields. Headers do not show any addresses that were included in the **BCC:** field, as these addresses were intended to remain private.
+- **Received:** Displays a sequential list of computer and servers that received this message, the time they received this message, and the final destination of the message. **Received** appears many times in a message header and should be read from bottom to top, as the first recipient is at the bottom of the header.
+- **Reply-To:** Determines which email address is auto-populated when you click the reply button to reply to an email in your email client. Spammers can easily fake **Reply-To** addresses. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest adding a closing slash at the end of this link.

File: content/exchange/view-and-read-email-headers-in-owa.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+- **Return-Path:** Like the **Reply-To:** address, this is where return mail is sent. Spammers can easily fake a return path. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
+- **Message-ID** A unique identifier assigned to a message. The **Message-ID** is useful for diagnosing a duplicate email issue. If you compare the **Message-ID** for multiple emails, and the IDs match, you know those messages are duplicates.
+- **X-Originating-Ip:** The IP address of the computer that sent the message. While this is slightly more difficult to fake, it is still possible. The originating IP address is typically the most reliable information about where the message actually came from. See [Email spoofing explained](/how-to/email-spoofing-best-practices) for guidance on detecting fraud email.
```
Comment (RackerKate) [inline content/exchange/view-and-read-email-headers-in-owa.md]: Suggest adding a closing slash at the end of this link.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4b7a0d1dad/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d0e757de6d/how-to/)

**Pull request merged**